### PR TITLE
Improved frontend of 404 page

### PR DESCRIPTION
**Pull request by KshitizSachan**
Added button on 404 page that leads back to the start page.

### CODE REVIEW

File: frontend/src/pages/404Page.js [hunk 1]
```diff
@@ -6,20 +6,42 @@ const PageNotFound = () => {
...
-        <h1>
+        <h1 style={{padding:"0px", }}>
           404  :) 
-		  <br/>
+          </h1>
+		  {/* <br/> */}
+      <h1>
 		  Page Not Found 
         </h1>
+        <a class="login-button" href="https://scholar-chat-orcid.herokuapp.com/">
```
Comment (chirag35847) [inline frontend/src/pages/404Page.js]: Please make it dymanic like this
const ENDPOINT = process.env.NODE_ENV==="production"?process.env.ENDPOINT:process.env.DEV_ENV;

**Comment by chirag35847** [comment]
Bhai dynamic banao

**Comment by chirag35847** [comment]
a tage me jo link hai, usse aise kro 
const ENDPOINT = process.env.NODE_ENV==="production"?process.env.ENDPOINT:process.env.DEV_ENV;

File: frontend/src/pages/404Page.js (hunk 1 above)
Comment (kuchiPie) [inline frontend/src/pages/404Page.js]: the button should send the user to the endpoint dynamically as mentioned by chirag in the comments

Comment (KshitizSachan) [reply frontend/src/pages/404Page.js]: ok

File: frontend/src/pages/404Page.js [hunk 2]
```diff
@@ -1,25 +1,34 @@
...
-        <h1>
+        <h1 style={{padding:"0px", }}>
           404  :) 
-		  <br/>
+          </h1>
+		  {/* <br/> */}
+      <h1>
 		  Page Not Found 
         </h1>
+        <a class="login-button" href="ENDPOINT">
```
Comment (kuchiPie) [inline frontend/src/pages/404Page.js]: It doesn't work like that bro...
See how to make a variable in react

Comment (kuchiPie) [reply frontend/src/pages/404Page.js]: Check if your code works locally

**Pull request merged**